frontend/public/models/index.ts

```diff
@@ -1,6 +1,19 @@
 // eslint-disable-next-line no-unused-vars
 import { K8sKind } from '../module/k8s';
 
+export const CatalogSourceConfigModel: K8sKind = {
```

Comment:
Remove this model if we aren't using it yet.

Reply:
I think we're using it now in the `Firehose`.